WebContent/AddBook.html

```diff
@@ -108,7 +109,7 @@ <h1>Welcome to Online Book Store</h1>
 					BookAuthor: <input type="text" name="author"><br/><br/>
 					Book Price: <input type="text" name="price"><br/><br/>
 					Book Qnty : <input type="text" name="quantity"><br/><br/>
-					<input type="submit" value=" Add These Books ">
+					<button type="button" class="btn">Add this book</button>

```

Comment:
Great work @pooranjoyb , but form submit is not working after this PR, can you check in local once

Reply:
Sure, Will check this out asap.

Reply:
@shashirajraja Hello, sir, should I continue working on this and post a pull request for the changes or you want this to be as it is.
Asking because the issue #64 that you assigned me

Reply:
Hi @pooranjoyb, the issue is resolved by other members and there will be conflicts in merging your PR too, it would be better if you take other issues.

Reply:
Thank you sir, I have found another issue that need to be fixed. Found that while working on the button issue. 
Would be glad if you assign that me when I open that :)

Reply:
that would be much better .there are many other issues if you are not able to find them i can help you in that .
if finding any difficulty can ask for help